src/main.ts

```diff
@@ -55,26 +50,11 @@ async function run(): Promise<void> {
       );
     }
 
-    // Add warning if using credentials
-    let credentialsJSON: Credential | undefined;
-    if (credentials) {
-      logWarning(
-        'The "credentials" input is deprecated. ' +
-          'Please switch to using google-github-actions/auth which supports both Workload Identity Federation and JSON Key authentication. ' +
-          'For more details, see https://github.com/google-github-actions/get-gke-credentials#authorization',
-      );
-
-      credentialsJSON = parseCredential(credentials);
-    }
-
     // Pick the best project ID.
```

Comment:
I think we can simplify this logic to `presence(core.getInput('project_id')) ||  presence(clusterName.projectID) || presence(process.env?.GCLOUD_PROJECT)`?

Reply:
We could for project ID and location but I am thinking the verbose logging could be helpful to debug issues?